api/src/main/java/javax/enterprise/inject/spi/builder/InjectionPointConfigurator.java

```diff
@@ -41,51 +37,6 @@
 public interface InjectionPointConfigurator {
 
     /**
-     * Read the InjectionPoint information from the given {@link Field}.
-     * All relevant information is overwritten.
-     *
-     * @param field defining the InjectionPoint.
-     * @return self
-     */
-    InjectionPointConfigurator read(Field field);
```

Comment:
@manovotn yes I agree. I wanted to go through the javadoc and spec text as well. I'll update. 

Additionaly I wanted to emphasize that it doesn't make sense to me to introduce such "implicit rule" for IPs. It would be just confusing and  brings next questions about its functionality (Does PIP get fired for this and etc.)